src/resources/cache/hashes.json

```diff
@@ -1,106 +1,106 @@
 {
```

Comment:
It makes sense that these hash files change but the binary files should not have. See the bold thing here: https://github.com/gpuweb/cts/blob/main/docs/case_cache.md#file-cache

If this is all it took to trigger this - and I should have realized it would be - then I think we're going to need to fix that issue sooner rather than later. @ben-clayton

Reply:
https://github.com/gpuweb/cts/pull/3187